accounts/usbwallet/wallet.go

```diff
@@ -514,7 +518,44 @@ func (w *wallet) SelfDerive(bases []accounts.DerivationPath, chain ethereum.Chai
 // signHash implements accounts.Wallet, however signing arbitrary data is not
 // supported for hardware wallets, so this method will always return an error.
 func (w *wallet) signHash(account accounts.Account, hash []byte) ([]byte, error) {
-	return nil, accounts.ErrNotSupported
+	w.stateLock.RLock() // Comms have their own mutex
```

Comment:
I guess this method should be renamed signData? Also please rename the hash variables everywhere to data.

Reply:
Please read the comment and update it too accordingly :P